R/utilities.R

```diff
@@ -840,10 +840,24 @@ track_recent_covid_increases <- function(
   num_fac = 5,
   overwrite_data = TRUE) {
   ## define inputs for data filtering
-  date <- Sys.Date()
-  state_df <- read_csv("https://raw.githubusercontent.com/uclalawcovid19behindbars/data/master/historical-data/historical_state_counts.csv")
   latest_scrape_date <-  max(scrape_df$Date)
   delta_start_date <- latest_scrape_date - lubridate::days(delta_days)
+  lookaround_delta_start_date <- c(delta_start_date, 
+                                   (delta_start_date + lubridate::days(1)),
+                                    (delta_start_date - lubridate::days(1)))
```

Comment:
To make this more flexible to running outside of the scraper schedule days, could we change to filter to something like `min(abs(delta_start_date - Date))`?

Reply:
I like this idea a lot!! Ran out of time to implement your suggestion today, but will do it ASAP on tuesday